app/styles/ui/_tutorial.scss

```diff
@@ -1,12 +1,117 @@
 .tutorial-panel-component {
+  display: flex;
+  flex-flow: column nowrap;
   min-width: 300px;
+  max-width: 350px;
+  background-color: var(--background-color);
+  box-shadow: var(--base-box-shadow);
+  z-index: var(--side-panel-z-index);
+  overflow-y: scroll;
+
+  .titleArea {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: var(--spacing-triple) var(--spacing-double) var(--spacing-double) var(--spacing-quint);
+
+    h1 {
+      margin: 0;
+      font-weight: var(--font-weight-light);
+    }
+    svg {
+      height: 55px;
+      width: 55px;
+    }
+  }
 
   ol,
   li,
   summary {
     list-style: none;
+    padding: 0;
+    margin: 0;
   }
   summary::-webkit-details-marker {
     display: none;
   }
+
+  li,
+  .titleArea {
+    border-bottom: var(--base-border);
+  }
+
+  li {
+    padding: var(--spacing-double);
+    padding-right: var(--spacing-three-halves);
```

Comment:
I feel a bit sad about having to introduce a new uneven spacer just for this. I can be convinced certainly but I really want to try to keep the number of spacers available to us to a minimum or else we'll lose the consistency real quick. I think if we drop this and stick with the double spacing and also drop the 6px padding around the `required-status-check` SVG it should all align quite nicely on the right hand side.

<img width="311" alt="image" src="https://user-images.githubusercontent.com/634063/64975447-5943c200-d8af-11e9-9639-55dbec5bcd8a.png">

Reply:
that sounds great! will do